samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -23,15 +23,15 @@ static void sensor_cli_data_cb(struct bt_mesh_sensor_cli *cli,
 		if (value->val1) {
 			printk("Presence detected\n");
 		} else {
-			printk("End of presence\n");
+			printk("No presence detected\n");
 		}
 	} else if (sensor->id ==
 		   bt_mesh_sensor_time_since_presence_detected.id) {
 		if (value->val1) {
```

Comment:
Could you please also add handling for special value 0xFFFF here (ref: GSS v5, section 3.219.2)? I think this was missed originally.

Reply:
Added